reporter/html.js

```diff
@@ -611,12 +611,14 @@ export function escapeText( s ) {
 		// having the HTML reporter actively render anything.
 		if ( qunit ) {
 
+			qunit.setAttribute( "role", "main" );
+
 			// Since QUnit 1.3, these are created automatically if the page
 			// contains id="qunit".
 			qunit.innerHTML =
 			"<h1 id='qunit-header'>" + escapeText( document.title ) + "</h1>" +
-			"<h2 id='qunit-banner'></h2>" +
-			"<div id='qunit-testrunner-toolbar'></div>" +
+			"<div id='qunit-banner'></div>" +
```

Comment:
I think this should wait until the QUnit 3.0 cycle. The other two changes are fine though, and should not affect compatibility in any way.

Reply:
That sounds fair, thanks for the review!